src/Compilers/VisualBasic/Portable/Semantics/Operators.vb

```diff
@@ -3404,6 +3404,12 @@ Next_i:
                 End Get
             End Property
 
+            Friend Overrides ReadOnly Property CallerArgumentExpressionParameterIndex As Integer
+                Get
+                    Return _parameterToLift.CallerArgumentExpressionParameterIndex
```

Comment:
>Return _parameterToLift.CallerArgumentExpressionParameterIndex [](http://example.com/codeflow?start=20&length=62)

Is this code path reachable? Do we have a test verifying corresponding behavior?

Reply:
@AlekseyTs Doesn't seem to be reachable to me. I changed the implementation to throw.